aidants_connect_common/lookups.py

```diff
@@ -15,6 +15,9 @@ def as_sql(self, compiler, connection):
             )
 
         sql, params = compiler.compile(self.lhs)
+        # sql expression par will appear 2 times in final expression so any parameters
+        # will need to be formatted twice.
+        params.extend(params)
```

Comment:
Ce changement est en rapport [avec la modification sur la validation des contraintes](https://docs.djangoproject.com/en/4.1/releases/4.1/#validation-of-constraints). Concrètement, appeler `.is_valid()` sur un `ModelForm` va déclancher une validation des contraintes SQL du modèle en même temps. Cependant, comme potentiellement l'objet n'existe pas encore en base, l'ORM génère un template SQL un petit peu différent ce qui nous oblige à être plus stricts ici.

Difficile d'expliquer la situation plus clairement sans montrer l'arrêt sur un breakpoint.